src/main/java/seedu/address/model/TaskManager.java

```diff
@@ -112,6 +112,14 @@ public void removeTask(Task key) {
         tasks.remove(key);
     }
 
+    /**
+     * Updates tasks that are overdue
+     */
+    public void checkOverdue() {
```

Comment:
The naming of this method implies that there are no side effects beyond checkingOverdue. Would this be better as `updateIfOverdue`?

Reply:
Sounds good